### ISSUE TITLE
Composer V2.0 Deprecation Warning for PSR-4 Namespace

### ISSUE THREAD

**Issue by DynamoGeek** [googleapis/gax-php#264]
Composer reports that a file in the Testing directory have namespaces that aren't PSR-4 compliant. This can be solved from the user side in the short term in various ways, but the long term fix will need to be making the namespaces PSR-4 compliant.
```
Class GPBMetadata\Google\ApiCore\Tests\Unit\Example ./vendor/google/gax/metadata/Google/ApiCore/Tests/Unit/Example.php
```

**Comment by ericsala** [comment]
Any solution on this issue? Thank you!

**Comment by yuuan** [comment]
in dev:

```json
    "autoload-dev": {
        "psr-4": {
            "Google\\ApiCore\\Dev\\": "dev/src",
            "Google\\ApiCore\\": "tests",
            "GPBMetadata\\": "metadata"
        }
    },
```
`vendor/google/gax/metadata/Google/ApiCore/Tests/Unit/Example.php` is `GPBMetadata\Google\ApiCore\Tests\Unit\Example`.

in no-dev:

```json
    "autoload": {
        "psr-4": {
            "Google\\ApiCore\\": "src",
            "GPBMetadata\\Google\\": "metadata"
        }
    },
```
`vendor/google/gax/metadata/Google/ApiCore/Tests/Unit/Example.php` is `GPBMetadata\Google\Google\ApiCore\Tests\Unit\Example`.

**Comment by bshaffer** [comment]
fixed in #278
